docs/bug-reports.md

```diff
@@ -0,0 +1,21 @@
+# Bug report guidelines
+
+Please include these elements in your bug report to make it easier for us to help you.
+
+* A brief title
+
+* An explanation of what you were expecting vs. the actual result
+
+* Steps to take in order to reproduce the buggy behavior
+
+* Whether you are using Zulip in production or in the development
+environment, and whether these are old versions
+
+* Whether you are using the web view or an app to access Zulip
```

Comment:
Changed this to `Whether you are using the web app, a desktop app or a mobile
device to access Zulip` -- does this work?